attributes/default.rb

```diff
@@ -1 +1,18 @@
-default['sysfs']['package'] = "sysfsutils"
+default['sysfs']['package'] = 'sysfsutils'
+default['sysfs']['service'] = 'sysfsutils'
+
+default['sysfs']['init_type'] = value_for_platform(
```

Comment:
would it not be better to do some detection rather than having to manage a list of versions?